docs/uml/classDiagrams/StorageComponent.puml

```diff
@@ -0,0 +1,67 @@
+@startuml
+!include ../style.puml
+
+skinparam classAttributeIconSize 0
+skinparam classFontsize 14
+skinparam classAttributeFontSize 13
+skinparam arrowThickness 2
+skinparam noteFontSize 13
+
+hide circle
+
+class UI {
+    + printDirectoryNotFoundMessage() : void
+    + printDirectoryCreatedMessage() : void
+    + printExerciseFileNotFoundMessage() : void
+    + printExerciseFileCreatedMessage() : void
+}
+
+class WerkIt
+
+package "Command Component" <<rectangle>> {
+    class WorkoutCommand
+    class PlanCommand
+    class ScheduleCommand
+}
```

Comment:
Changed naming of Command --> Logic component x)

Reply:
Informed via Telegram. Resolved in Commit 4e71996.